apps/docs/docs/user/intro/update.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 2
+title: Update Jayvee
+---
+
+# How to update Jayvee
+
+Jayvee is constently getting updates as a dedicated team of developers diligently addresses performance issues and incorporates new features to meet the evolving needs of its user base.
```

Comment:
Same here, some proofreading or automated help (I suggest languagetool). Also no need for marketing texts here ;).

```suggestion
Jayvee is consistently getting updates. To ensure you use the most recent version, please regularly update the interpreter and VSCode extension.
```